detection-rules/impersonation_github.yml

```diff
@@ -11,7 +11,7 @@ source: |
       or ilike(sender.email.email, '*github*')
       or iedit_distance(sender.email.domain.sld, 'github') <= 2
   )
-  and sender.email.domain.root_domain not in ('github.com', 'gitlab.com', 'itthub.net')
+  and sender.email.domain.root_domain not in ('github.com', 'gitlab.com', 'itthub.net', 'githubsupport.com')
```

Comment:
how confident are you in this? I don't get any results from a domain lookup

Reply:
hmmm, looks like it has MX and TXT records though?
https://www.whatsmydns.net/dns-lookup?query=githubsupport.com&server=google

Reply:
i've communicated with them before

<img width="281" alt="image" src="https://user-images.githubusercontent.com/11003450/181608175-28b01fa5-4e6c-48c5-bccc-4b30db5db5ad.png">

Reply:
okay, it's zendesk
https://webcache.googleusercontent.com/search?q=cache:QeIMJ5eQ5BIJ:https://github.community/t/emails-from-support-githubsupport-com/171928+&cd=1&hl=en&ct=clnk&gl=us

https://news.ycombinator.com/item?id=23703453